### PR TITLE
[Enhancement] Message/button should be more in line with design mockup when employee must have a "qualifying event"

### LINKED ISSUE #119: [Enhancement] Message/button should be more in line with design mockup when employee must have a "qualifying event"
Keep in mind this is called a "qualifying event" not a "life changing event". Please refer to https://www.sketch.com/s/09cd8050-fd53-4c45-a4f3-ccdef3efeae7/a/m4lxAL if you take this issue.

### PR DESCRIPTION
**Pull request by JavaLavaMT**
closes #119

Screen Shot 2020-09-18 at 9 39 41 AM
Screen Shot 2020-09-18 at 9 39 50 AM

### CODE REVIEW

File: app/controllers/account/benefits/index.js [hunk 1]
```diff
@@ -2,6 +2,20 @@ import Controller from '@ember/controller';
...
 export default class AccountBenefitsIndexController extends Controller {
+  qualifyingEvents = [
+    'Marriage',
+    'Declaration of Domestic Partnership',
+    'Death of Spouse or Dependent',
+    'Divorce',
+    'Annulment or Legal Separation',
+    'Birth, Adoption or Placement for Adoption',
+    'Loss of other coverage'
+  ];
```
Comment (James1x0) [inline app/controllers/account/benefits/index.js:13]: Move this to statics in config

File: app/templates/account/benefits/index.hbs [hunk 2]
```diff
@@ -49,12 +49,22 @@
             If you are eligible for benefits in the future, your company's next open enrollment period is
             {{time this.model.openEnrollment}}-{{time this.model.openEnrollment.end}}.
           {{/if}}
+          {{#if this.enrollment}}
+            In order to make changes, you mush have a qualifying event.
```
Comment (James1x0) [inline app/templates/account/benefits/index.hbs:53]:
```suggestion
            In order to make changes, you must have a qualifying event.
```

File: app/templates/account/benefits/index.hbs [hunk 3]
```diff
@@ -49,12 +49,22 @@
...
+            <a href="#">
+              <EmberPopover @side="right" @popperContainer="body">
```
Comment (James1x0) [inline app/templates/account/benefits/index.hbs:55]:
```suggestion
              <EmberTooltip @side="right" @popperContainer="body">
```

File: app/templates/account/benefits/index.hbs [hunk 4]
```diff
@@ -49,12 +49,22 @@
...
+                {{#each this.qualifyingEvents as |event|}}
+                  <div>{{event}}</div>
+                {{/each}}
+              </EmberPopover>
```
Comment (James1x0) [inline app/templates/account/benefits/index.hbs:59]:
```suggestion
              </EmberTooltip>
```

File: app/templates/account/benefits/index.hbs [hunk 5]
```diff
@@ -49,12 +49,22 @@
             If you are eligible for benefits in the future, your company's next open enrollment period is
             {{time this.model.openEnrollment}}-{{time this.model.openEnrollment.end}}.
           {{/if}}
+          {{#if this.enrollment}}
+            In order to make changes, you mush have a qualifying event.
+            <a href="#">
+              <EmberPopover @side="right" @popperContainer="body">
+                {{#each this.qualifyingEvents as |event|}}
+                  <div>{{event}}</div>
+                {{/each}}
```
Comment (James1x0) [inline app/templates/account/benefits/index.hbs:58]:
```suggestion
                <ul>
                  {{#each this.qualifyingEvents as |event|}}
                    <li>{{event}}</li>
                  {{/each}}
                </ul>
```

**Review by James1x0 — CHANGES_REQUESTED** [review]

File: app/templates/account/benefits/index.hbs (hunk 3 above)
Comment (JavaLavaMT) [reply app/templates/account/benefits/index.hbs:55]: when using tooltip the info seems to wrap

image

File: app/templates/account/benefits/index.hbs (hunk 5 above)
Comment (JavaLavaMT) [reply app/templates/account/benefits/index.hbs:58]: I had it with the list, but as you said  😃 
image

image

File: app/controllers/account/benefits/index.js (hunk 1 above)
Comment (JavaLavaMT) [reply app/controllers/account/benefits/index.js:13]: 🏁

File: app/templates/account/benefits/index.hbs (hunk 2 above)
Comment (JavaLavaMT) [reply app/templates/account/benefits/index.hbs]: 🏁

File: app/templates/account/benefits/index.hbs (hunk 5 above)
Comment (James1x0) [reply app/templates/account/benefits/index.hbs:58]: The difference is, this looks a lot better than the errors that were being displayed in that message.

Comment (James1x0) [reply app/templates/account/benefits/index.hbs:58]: Solidify the list! 😄

File: app/templates/account/benefits/index.hbs (hunk 3 above)
Comment (James1x0) [reply app/templates/account/benefits/index.hbs:55]: Less of a problem with the list 😏

**Review by James1x0 — CHANGES_REQUESTED** [review]
Still needs a list for qual events

**Pull request merged**